docs/implementation-guide.md

```diff
@@ -151,4 +146,150 @@ The table below describes the contents and expected ingestion pipeline behavior
 |VXU-V04-02_failedConversion.hl7| A single invalid VXU message that cannot be converted to FHIR.| The ingestion process will fail during the initial conversion to FHIR step. Information about the failure is written to `failed_fhir_conversion\vxu\`.
 |VXU-V04-02_failedUpload.hl7| A single VXU message that converts to an invalid FHIR bundle.| The ingestion pipeline will fail during the final step when it attempts to upload the data to the FHIR server. Information about the failure is written to `failed_fhir_uploads\vxu\`.|
 |VXU-V04-02_success_batch.hl7| A batch Hl7 message containing two valid VXU messages.| The ingestion pipeline is triggered twice and runs successfully to completion both times.|
-|VXU-V04-03_batch_1_success_1_failConversion.hl7| A batch Hl7 message containing one valid and one invalid VXU message.| The ingestion pipeline will run twice. On one execution it successfully process the data and uploads to the FHIR server. On the other execution it fails.|
\ No newline at end of file
+|VXU-V04-03_batch_1_success_1_failConversion.hl7| A batch Hl7 message containing one valid and one invalid VXU message.| The ingestion pipeline will run twice. On one execution it successfully process the data and uploads to the FHIR server. On the other execution it fails.|
+
+
+### Basic guide to FHIR Server REST calls:
+Once you have data in your FHIR server you can access it through REST API calls. Below are SOME examples of calls you can make.
+
+#### Prerequisites:
+- Access to Azure FHIR server
+
+- Azure cli tool
+
+- Bearer token (autoexpires and may need regeneration every 60~90 mins)
+
+    - Generate Token:
+
+      `az account get-access-token --resource=https://phdi-dev-fhir-server.azurehealthcareapis.com --query accessToken --output tsv`
+
+- Api tool - Postman, Insomnia, or curl. Make sure to set up your auth header using the generated token.
+    - Postman Example Auth Header key value pair: `Authorization: Bearer <Your Token>`
+    - The end of Step 4 of this guide has a CURL example
+
+#### Gotchas:
+Watch out for capitalization in urls, for example:
+{FHIR-server-url}/Patient not {FHIR-server-url}/patient
+
+GET Metadata, does not require auth, good for health checking the server
+`{{FHIR-server-url}}/Metadata`
+
+###Basic Templates
+####GET Requests
+Template format to GET all of a certain resource:  
+`{{FHIR-server-url}}/{{Resource Name}}`
+
+Examples:
+
+    GET all Patients:  
+    `{{FHIR-server-url}}/Patient`
+    
+    GET all immunizations:  
+    `{{FHIR-server-url}}/Immunization`
+
+
+Template to GET all resources associated with a specific resource:  
+`{{FHIR-server-url}}/{{Resource Name}}/<id>/{{Other Resource Name}}`
+
+Examples:
+
+    GET all Observations associated with a Patient ID:  
+    `{{FHIR-server-url}}/Patient/<PatientID>/Observation`
+    
+    Template format to GET a specific resource:  
+    `{{FHIR-server-url}}/{{Resource Name}}/<id>`
+
+    GET a specific Patient by id:  
```

Comment:
@gordonfarrell When you push up changes for this, can you bump this template out, and put it above line 191? I think it makes more sense to organize it that way.